raiden-dapp/tests/unit/app.spec.ts

```diff
@@ -38,15 +39,30 @@ describe('App.vue', () => {
         $t: (msg: string) => msg,
       },
     });
+  };
+
+  beforeEach(() => {
+    delete process.env.VUE_APP_IMPRINT;
```

Comment:
Suggestion: To make easier to see the full functionality, just add a `else` case in the `createWrapper` function to set this to `undefined` or delete it. But don't split both "setting" parts of this variable.

Reply:
Done!